apps/src/music/views/soundsPanel.module.scss

```diff
@@ -1,60 +1,87 @@
 @import "color.scss";
 
 .soundsPanel {
+  background-color: $neutral_dark;
   user-select: none;
-}
+  width: 300px;
+  height: 290px;
+  overflow-x: hidden;
+  overflow-y: auto;
 
-.folder {
-  background-color: $neutral_dark;
-  overflow: hidden;
-  border-radius: 4px;
-  padding: 6px;
-  &:first-child {
-    margin-top: 0;
-  }
-  &:not(:first-child) {
-    margin-top: 6px;
-  }
-}
+  .folderRow {
+    position: sticky;
+    top: 0;
+    padding: 6px 0;
+    margin-bottom: 6px;
+    background-color: $neutral_dark;
+    overflow: hidden;
 
-.folderName {
-  font-size: 18px;
-  padding: 6px 0;
-  margin-bottom: 6px;
-  border-bottom: solid 1px $neutral_dark90;
-  margin-right: 12px;
-}
+    &:first-child {
+      margin-top: 0;
+    }
+    &:not(:first-child) {
+      margin-top: 14px;
+    }
 
-.soundRow {
-  overflow: hidden;
-  padding: 4px 0;
-  &:hover {
-    background-color: $neutral_dark90;
-  }
-  &Selected {
-    background-color: $neutral_dark80;
+    &Left {
+      width: 65px;
+      height: 65px;
+      margin: 0 6px 0 4px;
+      float: left;
+    }
+
+    &Middle {
+      width: calc(100% - 75px - 58px);
+      float: left;
```

Comment:
Would it be easier to use flexbox here (rather than `float` + calculating width)?

Reply:
Oh good point.  I'll try to look into this on the next revision of this UI.